cmd/agent.go

```diff
@@ -71,14 +72,45 @@ func (ar AgentRunner) Run(cmd *cobra.Command, args []string) error {
 
 	defer ar.closePluginClients()
 
+	daemon, err := cmd.Flags().GetBool("daemon")
+
+	if err != nil {
+		return err
+	}
+
+	if daemon == true {
+		for {
+			err := ar.runInstance(plugins, policyBundles)
+
+			if err != nil {
+				ar.logger.Error("error running instance", "error", err)
+			}
+
+			time.Sleep(time.Second * 60)
+		}
+	} else {
+		err := ar.runInstance(plugins, policyBundles)
+
+		if err != nil {
+			ar.logger.Error("error running instance", "error", err)
```

Comment:
```suggestion
			ar.logger.Error("error running instance", "error", err)
			return err
```

Reply:
I thought about this one a bit and wasn't (and still am) unsure as to the best way to handle this. With this method you'll get a retry and that seems better to me in daemon mode.

Reply:
Fixed and added a comment above to remind us to think about retries later.